homeassistant/components/system_bridge/sensor.py

```diff
@@ -0,0 +1,346 @@
+        return self._device_class
+
+    @property
+    def unit_of_measurement(self) -> str:
+        """Return the unit this state is expressed in."""
+        return self._unit_of_measurement
+
+
+class BridgeBatterySensor(BridgeSensor):
+    """Defines a Battery sensor."""
+
+    def __init__(self, coordinator: DataUpdateCoordinator, bridge: Bridge):
+        """Initialize System Bridge sensor."""
+        super().__init__(
+            coordinator,
+            bridge,
+            "battery",
+            "Battery",
+            None,
+            DEVICE_CLASS_BATTERY,
+            PERCENTAGE,
+        )
+
+    @property
+    def state(self) -> float:
+        """Return the state of the sensor."""
+        bridge: Bridge = self.coordinator.data
+        return bridge.battery.percent
+
+    @property
+    def device_state_attributes(self) -> Optional[Dict[str, Any]]:
+        """Return the state attributes of the entity."""
+        bridge: Bridge = self.coordinator.data
+        return {
+            ATTR_CAPACITY_MAX: bridge.battery.maxCapacity,
+            ATTR_CAPACITY: bridge.battery.currentCapacity,
+            ATTR_CHARGING: bridge.battery.isCharging,
+            ATTR_MANUFACTURER: bridge.battery.manufacturer,
+            ATTR_MODEL: bridge.battery.model,
+            ATTR_SERIAL: bridge.battery.serial,
+            ATTR_TIME_REMAINING: bridge.battery.timeRemaining,
+            ATTR_TYPE: bridge.battery.type,
+            ATTR_VOLTAGE: bridge.battery.voltage,
+        }
+
+
+class BridgeCpuSpeedSensor(BridgeSensor):
+    """Defines a CPU sensor."""
+
+    def __init__(self, coordinator: DataUpdateCoordinator, bridge: Bridge):
+        """Initialize System Bridge sensor."""
+        super().__init__(
+            coordinator,
+            bridge,
+            "cpu_speed",
+            "CPU Speed",
+            None,
+            None,
+            FREQUENCY_GIGAHERTZ,
+        )
+
+    @property
+    def state(self) -> float:
+        """Return the state of the sensor."""
+        bridge: Bridge = self.coordinator.data
+        return bridge.cpu.currentSpeed.avg
+
+    @property
+    def device_state_attributes(self) -> Optional[Dict[str, Any]]:
+        """Return the state attributes of the entity."""
+        bridge: Bridge = self.coordinator.data
+        return {
+            ATTR_BRAND: bridge.cpu.cpu.brand,
+            ATTR_CORES_PHYSICAL: bridge.cpu.cpu.physicalCores,
+            ATTR_CORES: bridge.cpu.cpu.cores,
+            ATTR_GOVERNOR: bridge.cpu.cpu.governor,
+            ATTR_MANUFACTURER: bridge.cpu.cpu.manufacturer,
+            ATTR_SPEED_CURRENT_MAX: bridge.cpu.currentSpeed.max,
+            ATTR_SPEED_CURRENT_MIN: bridge.cpu.currentSpeed.min,
+            ATTR_SPEED_MAX: bridge.cpu.cpu.speedMax,
+            ATTR_SPEED_MIN: bridge.cpu.cpu.speedMin,
+            ATTR_SPEED: bridge.cpu.cpu.speed,
+            ATTR_VENDOR: bridge.cpu.cpu.vendor,
+            ATTR_VOLTAGE: bridge.cpu.cpu.voltage,
```

Comment:
For all of the measurements in this platform, please make a separate sensor for each non static measurement. Remove the static info.

Reply:
I've cleared out most of the attributes and made a couple of more sensors from these.

I've left the filesystem and the load ones. These can be useful, and only change when state changes, but I don't think they need a dedicated sensor at this point.

There are a few more to add from recent releases, but I'll leave those until future additions :smiley_cat: